.eslintrc.json

```diff
@@ -26,15 +26,18 @@
         "ExportDeclaration": { "minProperties": 2 }
       }
     ],
-    "comma-dangle": [2, {
-      "arrays": "always-multiline",
-      "objects": "always-multiline",
-      "imports": "never",
-      "exports": "always-multiline",
-      "functions": "always-multiline"
-    }],
+    "comma-dangle": [
+      2,
+      {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "never",
+        "exports": "always-multiline",
+        "functions": "always-multiline"
+      }
+    ],
     "import-newlines/enforce": "warn",
-    "newline-destructuring/newline": "warn",
+    "newline-destructuring/newline": ["warn", { "items": 1 }],
```

Comment:
If you do changes like that in a PR that should only touch other files, then you can easily create more work and conflicts, which is the case now. For the future, please separate PRs like that so we will not have conflicts.

Reply:
Okay